app/build.gradle

```diff
@@ -164,3 +164,11 @@ task copyTextFiles(type: Copy) {
 }
 
 preBuild.dependsOn filterAbout, copyTextFiles
+
+allprojects {
+    gradle.projectsEvaluated {
+        tasks.withType(JavaCompile) {
+            options.compilerArgs << "-Xlint" << "-Xlint:-deprecation" << "-Xlint:-serial"
```

Comment:
This is something that should be documented. (What is the rationale behind these arguments?)